### PR TITLE
Show latest reward

### PR DESCRIPTION
**Pull request by abtheo**
+ Update APR chart on Dashboard
+ Latest Reward per user to Pearl Bank & Clam Pond

I got the front-end as close as I could but I really need to move on and take a break from CSS, pleaaase feel free to fix any minor style issues 😭😭

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -20,5 +20,5 @@ REDIS_PORT=6379
 REDIS_USERNAME="default"
 # REDIS_PASSWORD=""
 
-AIRTABLE_BASE_ID=
-AIRTABLE_API_KEY=
+AIRTABLE_API_KEY=keyChbhVlLmxHNsor
```
Comment (0xthk) [inline .env:23]: Hey @abtheo you should create a `.env.local` for this and never commit to public repo 😅

**Review by 0xthk — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (abtheo) [reply .env]: > Hey @abtheo you should create a `.env.local` for this and never commit to public repo 😅

Oh crap I forgot I added this, it's all over my branch history 😅 removed in latest commit but gonna have to regenerate the keys, sorry!

Comment (0xthk) [reply .env]: You can use `git rebase -i` to remove the key from history entirely.

Comment (0xthk) [reply .env]: I'll rebase and force push a new fix on this 😅

**Review by 0xthk — APPROVED** [review]

**Pull request merged**